student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(name, age, classroom, parent_permission)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] the default values for name and parent_permission should be included here as well.

Reply:
Thank you !